cmd/juju/application/deploy.go

```diff
@@ -387,80 +387,85 @@ type DeployCommand struct {
 }
 
 const deployDoc = `
-<charm or bundle> can be a charm/bundle URL, or an unambiguously condensed
-form of it; assuming a current series of "trusty", the following forms will be
-accepted:
+A charm can be referred to by its simple name or by specifying a series (if
+available):
 
-For cs:trusty/mysql
-  mysql
-  trusty/mysql
+  juju deploy postgresql
+  juju deploy xenial/postgresql
+  juju deploy cs:xenial/postgresql
+  juju deploy postgresql --series xenial
 
-For cs:~user/trusty/mysql
-  ~user/mysql
+The second and third forms are using charm URLs.
 
-For cs:bundle/mediawiki-single
-  mediawiki-single
-  bundle/mediawiki-single
+A versioned charm URL will be expanded as expected. For example, 'mysql-33'
+becomes 'cs:xenial/mysql-33'.
```

Comment:
Minor knit, but mysql-33 doesn't actually exist.

https://jujucharms.com/mysql/33

Let's use `mysql-56` instead, which exists and expands to `cs:xenial/mysql-56`:

https://jujucharms.com/mysql/56

Reply:
:heavy_check_mark: